service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorServiceImpl.java

```diff
@@ -176,9 +180,23 @@ private void validateAdapterEntityOrFail(final AttributeAdapterConnection adapte
         if (adapter.getAdapterEndpoint() == null || adapter.getAdapterEndpoint().isEmpty()) {
             throw new AttributeConnectorException("Attribute adapter configuration requires a nonempty endpoint URL");
         }
+        try {
+            if (!new URI(adapter.getAdapterEndpoint()).getScheme().equalsIgnoreCase(HTTPS)) {
```

Comment:
The function will be like:
boolean isHttpsUrl(String urlStr) throws  URISyntaxException{
    if( (urlStr != null) && (!urlStr.isEmpty())){
        URI uri = URI(urlStr);
        String schemeUri = url.getScheme();
        if(schemeUri == null || (schemeUri.isEmpty()))
           return false;
          //i dont like the name HTTPS so changing into to HTTPS_STR
       return (HTTPS_STR.equals(schemeUri) == true) ? true : false)
  }

   return false;
}